subprojects/gradle/build-metrics/src/main/kotlin/com/avito/android/plugin/build_metrics/internal/BuildOperationsResult.kt

```diff
@@ -0,0 +1,20 @@
+package com.avito.android.plugin.build_metrics.internal
+
+internal class BuildOperationsResult(
+    val tasksExecutions: List<TaskExecutionResult>,
+    val cacheOperations: CacheOperations
+)
```

Comment:
Introduced a model for build results based on operations.
Clients consume it at the end of a build.